lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.html

```diff
@@ -3,12 +3,45 @@
                    [multi]="multi">
         <mat-expansion-panel *ngIf="displayDefaultProperties"
                              [expanded]="canExpandProperties()"
-                             [attr.data-automation-id]="'adf-metadata-group-properties'">
+                             [attr.data-automation-id]="'adf-metadata-group-properties'"
+                             hideToggle
+                             (opened)="generalTogglePanelState()"
+                             (closed)="generalTogglePanelState()">
             <mat-expansion-panel-header>
-                <mat-panel-title class="adf-metadata-properties-title">
-                    {{ 'CORE.METADATA.BASIC.HEADER' | translate }}
-                </mat-panel-title>
+                <div class="adf-metadata-properties-panel-content">
+                    <mat-icon>
+                        {{ generalInfoPanelState ? 'expand_more' : 'chevron_right'}}
+                    </mat-icon>
+                    <mat-panel-title class="adf-metadata-properties-title">
+                        {{ 'CORE.METADATA.BASIC.HEADER' | translate }}
+                    </mat-panel-title>
+                </div>
+                <button *ngIf="!editable && !readOnly && hasAllowableOperations()"
```

Comment:
We should avoid calling functions in html, can you please use here some variable instead of function?

Reply:
@Yasa-Nataliya you liked it but I can't see changes for that. There is still usage of hasAllowableOperations function in html instead of using variable in html.